manuscript.md

```diff
@@ -482,23 +479,13 @@ this extensive list of possible metrics is through the use of ROC
 (receiver-operating-characteristic; False Positive Rate on the
 x-axis, and True Positive Rate on the y-axis) and PR (precision-recall;
 True-Positive-Rate on the x-axis, Positive-predictive-value on the y-axis)
-curves. These curves are generated by considering a continuum of thresholds
+curves (see @fig:example). These curves are generated by considering a continuum of thresholds
 of classifier acceptance, and computing the values of ROC/PR metrics for
 each value of the threshold. The area-under-the-curve (AUC) is then used as
 a validation metric and are typically called AUC-ROC (Area-Under-the-Curve
 Receiver-Operator-Curve) and AUC-PR (Area-Under-the-Curve Precision-Recall)
 (_e.g._ ROC-AUC in @tbl:validation).
 
-Note that for both plots in @fig:auc, the area-under-the-curve (AUC) of the
-random model is $0.5$, and the AUC of the perfect classifier is $1.0$. This
```

Comment:
I think we should keep the 0.5 is random and 1 is perfect sentiment. Just for the wise and the unwise

Reply:
this could go in the table, currently just says success -> 1 at the moment but could indicate 0.5 is the min

Reply:
true it is in the table so we can then just keep as is (i.e. all removed)